src/js/actions/SettingsActions.js

```diff
@@ -127,6 +127,15 @@ const refreshProgress = (percentage) => {
     });
 };
 
+const toggleDisplayNotifications = (value) => {
+    app.config.set('displayNotifications', value);
+    app.config.saveSync();
```

Comment:
There's an async method too `.save(callback)`.

It's fine for now as we don't do massive blocking stuff.